### PR TITLE
Added the Sort Maps operation

### PR DESCRIPTION
**Pull request by IuliaMoldovan**
- Added the Sort Maps Action and Service

- Changed the description for the Get Keys Action

### CODE REVIEW

File: cs-maps/src/main/java/io/cloudslang/content/maps/actions/SortMapsAction.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     * @param sortOrder        The order in which the selected entries will be sorted.
+     *                         Valid values: asc (ascending), desc (descending).
+     * @param pairDelimiter    The separator to use for splitting key-value pairs into key, respectively value.
+     *                         Default value: ':'.
+     *                         Valid values: Any value that does not contain or is equal to entry_delimiter.
+     * @param entryDelimiter   The separator to use for splitting the map into entries.
+     *                         Valid values: Any value.
+     * @param mapStart         Optional - A sequence of 0 or more characters that marks the beginning of the map.
+     *                         Valid values: Any value.A sequence of 0 or more characters that marks the beginning of the map.
+     *                         Valid values: Any value.
```
Comment (ericandreica) [inline cs-maps/src/main/java/io/cloudslang/content/maps/actions/SortMapsAction.java:51]: It's duplicate.

File: cs-maps/src/main/java/io/cloudslang/content/maps/utils/SortMaps.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                marker++;
+            }
+        }
+        return chunk.toString();
+    }
+
+    public int compare(String s1, String s2) {
+        if ((s1 == null) || (s2 == null)) {
+            return 0;
+        }
```
Comment (ericandreica) [inline cs-maps/src/main/java/io/cloudslang/content/maps/utils/SortMaps.java:38]: This does not mean that is s1 or s2 is null the comparator will consider them equals?

**Review by ericandreica — CHANGES_REQUESTED** [review]

File: cs-maps/src/main/java/io/cloudslang/content/maps/actions/SortMapsAction.java (hunk 1 above)
Comment (IuliaMoldovan) [reply cs-maps/src/main/java/io/cloudslang/content/maps/actions/SortMapsAction.java:50]: fixed

**Review by ericandreica — APPROVED** [review]
Good job.

**Review by florinAndrei1991 — APPROVED** [review]

**Pull request merged**